### PR TITLE
Added common headers required by the API

### PR DESCRIPTION
**Pull request by skazi0**

### CODE REVIEW

**Comment by skazi0** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;lqn8yLQ0&#x2F;122-2-21-add-on-product-repo-checks-service-integration"> (2) #21 Add-On Product Repo Checks: Service Integration</a>

File: assets/app/data/services/upgrade/common-api.constants.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(function() {
+
+    angular
+        .module('crowbarData.upgrade')
+        .constant('COMMON_API_HEADERS', {
```
Comment (santiph) [inline assets/app/data/services/upgrade/common-api.constants.js]: Could you add the API version?
`COMMON_API2_HEADERS`

Comment (santiph) [reply assets/app/data/services/upgrade/common-api.constants.js]: Also on the file name it'd be good to update it

**Review by santiph — CHANGES_REQUESTED** [review]
Good implementation, please address the name change before merging

File: assets/app/data/services/upgrade/common-api.constants.js (hunk 1 above)
Comment (skazi0) [reply assets/app/data/services/upgrade/common-api.constants.js]: done.

**Pull request merged**